--- /usr/share/X11/xkb/rules/evdev.lst.orig    2022-08-08 09:44:44.774692664 +0200
+++ /usr/share/X11/xkb/rules/evdev.lst    2022-08-08 09:45:42.386812148 +0200
@@ -488,6 +488,7 @@
   mac             de: German (Macintosh)
   mac_nodeadkeys  de: German (Macintosh, no dead keys)
   dsb             de: Lower Sorbian
+  hsb             de: Upper Sorbian
   dsb_qwertz      de: Lower Sorbian (QWERTZ)
   qwerty          de: German (QWERTY)
   tr              de: Turkish (Germany)
