--- /usr/share/X11/xkb/symbols/de.orig    2022-08-08 09:29:08.004848445 +0200
+++ /usr/share/X11/xkb/symbols/de    2022-08-08 09:36:00.557594119 +0200
@@ -53,6 +53,16 @@
 };
 
 partial alphanumeric_keys
+xkb_symbols "hsb" {
+    // previous standard German layout with dead_caron mapped onto tilde
+
+    include "de(basic)"
+    name[Group1]="Upper Sorbian";
+
+    key <TLDE>    { [dead_caron, degree,    U2032,    U2033    ] };
+};
+
+partial alphanumeric_keys
 xkb_symbols "nodeadkeys" {
 
     // modify the basic German layout to not have any dead keys

